src/operator/src/insert.rs

```diff
@@ -580,6 +580,10 @@ impl Inserter {
                 // for it's a very unexpected behavior and should be set by user explicitly
                 for mut create_table in create_tables {
                     if create_table.table_name == trace_services_table_name(trace_table_name) {
+                        // Disable append mode for trace services table since it requires upsert behavior.
+                        create_table
+                            .table_options
+                            .insert(APPEND_MODE_KEY.to_string(), "false".to_string());
```

Comment:
Will it be a breaking change? Does it matter if users already have the service name table.

Reply:
Yes. It's a breaking change. Let me add the `!`.